Feature/22_LOGIN_VIEW

Symphonie234 opened the pull request:
- Added CSS to the login view.
- Created a new CSS file called "Login.css" to implement a new technique in using CSS.
- Also used the same tactic above but for the page title.
- Google Login is functional.

Symphonie234 committed:
added cosmetics and functionality to the login page

Symphonie234 committed:
Added functionality to google and cosmetics for the login page, also added css and avatar for the navigation bar

ChronoDevs commented on composer.lock:
don't push composer.lock

Symphonie234 replied:
Understood

ChronoDevs commented on resources/views/auth/login.blade.php:
```diff
@@ -1,9 +1,10 @@
     <div class="row justify-content-center">
-        <div class="col-md-8">
+        <div class="col-md-5">
             <div style="display: flex; justify-content: center; align-items: center;">
```
don't add style inside the html tag

Symphonie234 replied:
Understood

ChronoDevs commented on public/js/scripts.js:
```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", function() {
```
use single quote

ChronoDevs commented on resources/views/auth/login.blade.php:
```diff
@@ -1,9 +1,10 @@
+        <div class="col-md-5">
             <div style="display: flex; justify-content: center; align-items: center;">
                 <div class="card-body" id="card-body">
                     <div class="intro" style="text-align: center;">
```
don't add style inside the html tag

ChronoDevs commented on resources/views/auth/login.blade.php:
```diff
@@ -19,39 +20,50 @@
                     </div>
                     <br>
                     <div class="social-login" style="text-align: center;">
```
don't add style inside the html tag

ChronoDevs commented on resources/views/auth/login.blade.php:
```diff
@@ -19,39 +20,50 @@
+                            Continue with Facebook
+                        </a>
+                    </div>
+                    <div class="social-login" style="text-align: center;">
```
don't add style inside the html tag

ChronoDevs commented on config/services.php:
```diff
@@ -30,4 +30,15 @@
         'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
     ],
 
+    'facebook' => [
```
commit env,example also

Symphonie234 committed:
adjusted to the changes in the PR and added additional css to the buttons in the login page

Symphonie234 commented:
- removed styling in the HTML
- changed double quotes (" ") in the js folder into single quotes (' ')
- removed the composer.lock
- added content in the .env.example file

ChronoDevs commented on database/migrations/2014_10_12_000000_create_users_table.php:
```diff
@@ -26,9 +26,11 @@ public function up()
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->rememberToken();
+            $table->rememberToken()a;
```
a is created?

Symphonie234 committed:
fixed formatting in the users table

Symphonie234 commented:
- fixed the extra 'a' in the users table.
